src/dash/models/DashManifestModel.js

```diff
@@ -566,14 +566,20 @@ function DashManifestModel() {
         }
     }
 
+    function getEssentialPropertiesForAdaptation(adaptation) {
+        if (!adaptation || !adaptation.hasOwnProperty(DashConstants.ESSENTIAL_PROPERTY) || !adaptation.EssentialProperty.length) return [];
+        return adaptation.EssentialProperty.map( supp => {
```

Comment:
Rename `supp` to `essentialProperty`

Reply:
I used the same name `essentialProperty` for the subsequent Function (where just `prop`was used)